crates/solidity/outputs/npm/tests/src/errors.ts

```diff
@@ -8,11 +8,11 @@ test("render error reports", () => {
   const { errors } = language.parse(ProductionKind.SourceUnit, source);
   expect(errors).toHaveLength(1);
 
-  const report = errors[0]?.toErrorReport("test.sol", source, /* withColor */ false);
+  let reports = errors.map((error: any) => error.toErrorReport("test.sol", source, /* withColor */ false));
```

Comment:
I wonder why the change here? we already asserted it has a length of 1 above. Also, why would we use `any` here?

Reply:
> because the type was not inferred but now I see that it's unresolved in Node.js 

Is the generated `.d.ts` is broken somehow? not blocking to this PR, but I would have expected the earlier test to fail if it cannot resolve the types, since we compile with `struct: true`?